Reference/google/protobuf/unittest_mset.pb.swift

```diff
@@ -311,11 +295,35 @@ extension Proto2WireformatUnittest_TestMessageSet {
   }
 }
 
+/// A `SwiftProtobuf.SimpleExtensionMap` that includes all of the extensions defined by
+/// this .proto file. It can be used any place an `SwiftProtobuf.ExtensionMap` is needed
+/// in parsing, or it can be combined with other `SwiftProtobuf.SimpleExtensionMap`s to create
+/// a larger `SwiftProtobuf.SimpleExtensionMap`.
 let ProtobufUnittest_UnittestMset_Extensions: SwiftProtobuf.SimpleExtensionMap = [
   ProtobufUnittest_TestMessageSetExtension1.Extensions.message_set_extension,
   ProtobufUnittest_TestMessageSetExtension2.Extensions.message_set_extension
 ]
 
+extension ProtobufUnittest_TestMessageSetExtension1 {
+  struct Extensions {
```

Comment:
btw - since this is just a namespace for these, should we make it an `enum` instead of a `struct`?  (Since I think we did that in spots within the library for style)